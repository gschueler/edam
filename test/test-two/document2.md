% Document2

2nd document