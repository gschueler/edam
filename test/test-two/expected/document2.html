<!DOCTYPE html>
<html>
<head>
  <title>Document2</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="document2.html">Chapter 2: Document2</a></li>
        <li class="toc"><a href="index.html">Table of Contents</a></li>
        <li class="prev"><a href="document.html">Chapter 1: Document</a></li>
        
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Document2</h1>
<p
>2nd document</p
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="document2.html">Chapter 2: Document2</a></li>
        <li class="toc"><a href="index.html">Table of Contents</a></li>
        <li class="prev"><a href="document.html">Chapter 1: Document</a></li>
        
    </ul>
</nav>

</body>
</html>